### ISSUE TITLE
Rechnungsnummern zurücksetzen

### ISSUE THREAD

**Issue by Mucki526** [jlawyerorg/j-lawyer-org#2203]
Ich abe versehentlich eine Rechnungsnummer vergeben, die ich eigentlich bei einer anderen Akte nutzen wollte und die Rechnung schon verschickt hatte (VKH-Abrechnung, bekommt bei uns Rechnungsnummer). Also wollte ich den Beleg löschen und die Rechnungsnummer im Belegnummernkreis zurücksetzen. Hat geklappt, aber, als ich den richtigen Beleg erstellen wollte kam dieser Fehler:

![grafik](https://github.com/jlawyerorg/j-lawyer-org/assets/37142896/1032c7d8-606f-448d-b4dc-4dca95bf33a9)

**Comment by j-dimension** [comment]
Geht es um die letzte erstellte Rechnungsnummer, oder irgendetwas in der Mitte?

**Comment by Mucki526** [comment]
ja, bin dahinter gekommen, der Fehler kommt, wenn man versehentlich eine neue Start-Rechnungsnummer eingegeben hat, wo der Beleg noch existiert. Dann gibt es den Fehler. Das könnte abgefangen werden, wenn bei "Reset" des Startpunktes des Rechnungsnummernkreises, darauf hingewiesen wird, dass der neue Startpunkt als Beleg schon in der Akte... existiert. Bitte höheren Wert wählen..."

**Comment by j-dimension** [comment]
Ich fände es in solchen Fällen besser, den falschen Beleg auf storniert zu setzen. Er hat ja mal existiert.

**Comment by Mucki526** [comment]
Der Beleg, der in Finanzen erstellt wird, ist noch keine Rechnung, nur eine Buchung, hat dann aber eine Rechnungsnummer.. wenn er gelöscht wird. Anders sieht es aus, wenn der Buchung schon ein Rechnungsdokument zugeordnet wurde Und natürlich erst recht,wenn dieses versendet wurde. Aus Gründen der Übersichtlichkeit wäre es sinnvoll wenn die Rechnungsnummern fortlaufend sind.
